### PR TITLE
Unit 6 steady state updates

### PR DESCRIPTION
**Pull request by aspiece**
Lesson grammar, final project documents updates.
Project file updates

### CODE REVIEW

**Comment by aspiece** [comment]
Lesson, lab and project grammar, slide deck alignment, document updates, project document updates for Units 1, 2, 3 (oops, I forgot), Final Project document updates, and updated changelog.

File: docs/lesson_61.md.html [hunk 1]
```diff
@@ -42,75 +33,66 @@
...
 
-Demonstrate a few example projects (with as much variety as possible).
-  
-* Try to hit a bunch of different types of scripts.  Many students will gravitate towards games, but other options include simulations, productivity tools, musical projects, animations, and more.
+Explain that, for their final project, the students will get to design and build a script of their own choosing. Point out that this will involve more than just writing script. There will be planning, design, scheduling, and other project management tasks. Emphasize that students will be graded on not only the script they produce, but the process they used to design, plan, and implement it.
 
-Distribute the project rubric and point out key aspects of the requirements.
+Demonstrate a few example projects (with as much variety as possible).mTry to hit a bunch of different types of scripts. Many students will gravitate towards games, but other options include simulations, productivity tools, musical projects, animations, and more.
```
Comment (NagmaM) [inline docs/lesson_61.md.html:50]: Remove "m" before the word "try"

File: docs/lesson_61.md.html [hunk 2]
```diff
@@ -42,75 +33,66 @@
...
-* Briefly list a bunch of the things they've learned (drawing, animation, variables, loops, conditionals, loops, etc.)
-* Show a lab or assignment from early on and remind them that, not that long ago, this was challenging, whereas it now seems nearly trivial (hopefully).
+## Instructor's notes
 
-Explain that, for their final project, the students will get to design and build a script of their own choosing.
+### Introduce project
 
-* Point out that this will involve more than just writing script-* there will be planning, design, scheduling, and other project management tasks.
-* Emphasize that students will be graded on not only the script they produce, but the process they used to design, plan, and implement it.
+Talk about how far students have come this semester.mAsk students to think back to the start of the semester and remember how little they knew about scripting. Briefly list a bunch of the things they've learned (drawing, animation, variables, loops, conditionals, loops, etc.). Show a lab or assignment from early on and remind them that, not that long ago, this was challenging, whereas it now seems nearly trivial.
```
Comment (NagmaM) [inline docs/lesson_61.md.html:46]: Remove "m" before the word "ask"

File: docs/lesson_61.md.html [hunk 3]
```diff
@@ -42,75 +33,66 @@
...
 
-Distribute the project rubric and point out key aspects of the requirements.
+Demonstrate a few example projects (with as much variety as possible).mTry to hit a bunch of different types of scripts. Many students will gravitate towards games, but other options include simulations, productivity tools, musical projects, animations, and more.
 
-* Point out the steps in the process and that each one is equally important.
-* Specifically mention the large number of points for things _not_ related to coding.
-* Remind students that, as they are now experts, there are high expectations for the depth, complexity, and completeness of their projects.
+Distribute the project rubric and point out key aspects of the requirements. Point out the steps in the process and that each one is equally important Specifically mention the large number of points for things _not_ related to coding. Remind students that, as they are now experts, there are high expectations for the depth, complexity, and completeness of their projects.
```
Comment (NagmaM) [inline docs/lesson_61.md.html:52]: Add period after the word "important"

File: lesson_61.md [hunk 4]
```diff
@@ -34,73 +25,64 @@ Software development is a process.  It begins with carefully thinking about, and
...
-* Briefly list a bunch of the things they've learned (drawing, animation, variables, loops, conditionals, loops, etc.)
-* Show a lab or assignment from early on and remind them that, not that long ago, this was challenging, whereas it now seems nearly trivial (hopefully).
+## Instructor's notes
 
-Explain that, for their final project, the students will get to design and build a script of their own choosing.
+### Introduce project
 
-* Point out that this will involve more than just writing script-* there will be planning, design, scheduling, and other project management tasks.
-* Emphasize that students will be graded on not only the script they produce, but the process they used to design, plan, and implement it.
+Talk about how far students have come this semester.mAsk students to think back to the start of the semester and remember how little they knew about scripting. Briefly list a bunch of the things they've learned (drawing, animation, variables, loops, conditionals, loops, etc.). Show a lab or assignment from early on and remind them that, not that long ago, this was challenging, whereas it now seems nearly trivial.
```
Comment (NagmaM) [inline lesson_61.md:38]: remove "m" before "ask"

**Review by NagmaM — APPROVED** [review]

**Pull request merged**